tests/framework/db/mssql/QueryBuilderTest.php

```diff
@@ -85,9 +86,8 @@ public function testCommentColumn()
 
     public function testCommentTable()
     {
-        if ($this->driverName === 'sqlsrv') {
-            $this->markTestSkipped('Should be fixed');
-        }
+        // related to https://github.com/yiisoft/yii2/pull/17364
+        $this->markTestSkipped('Should be fixed');
```

Comment:
Looks like tests for non-MSSQL got disabled. Is that intentional?

Reply:
No, that condition was unnecessary, it's already MSSQL specific test class.